alacritty_macos.yml

```diff
@@ -232,6 +232,11 @@ mouse:
 
 selection:
   semantic_escape_chars: ",│`|:\"' ()[]{}<>"
+  # When set to true, selection string will be copy to primary clipborad.
+  # Default value is false, the selection string will be copy the selection
+  # clipborad.
+  save_to_clipboard: true
+
```

Comment:
For the macOS documentation the same issues mentioned for the linux configuration file apply.

However since macOS does not have a selection clipboard as far as I know, the documentation should be changed to reflect that.

There's also an additional newline at the end of the block which needs to be removed.